pkgs/top-level/python-packages.nix

```diff
@@ -4269,6 +4273,13 @@ in {
 
   mistune = callPackage ../development/python-modules/mistune { };
 
+  mistune_2 = callPackage ../development/python-modules/mistune {
+    source-info = {
+      version = "2.0.0a4";
+      sha256 = "0i6cblmjl58kdmaa21xm0l1ls0kvjpfy45sf73fw3ws6305f628k";
```

Comment:
I think it would be clearer if we converted the current default into a generic nix and created a 0.8.nix and a 2.nix file.

Reply:
Did it same way as postgresql do.